### ISSUE TITLE
Could you add options for alternate stores?

### ISSUE THREAD

**Issue by NXX99** [red-stone-network/clockwork#74]
Hi. I'm working on an app for Clockwork that's like the Windows 8 Desktop app for Clockwork. The problem: Cisco Umbrella has blocked **EVERY SINGLE VERCEL.APP URL!** Also, I can't use LTBEEF or anything since I'm on CROS 120. 

You can add a panel in the settings menu that would set the URL of the store, or have the store frontend be an iframe thing with an options bar up at the top to switch store URLs

I can't think of alternatives aside from forking.

Here's a screenshot of Cisco umbrella acting Special Ed when I try to go to the store.

![Screenshot 2024-02-07 8 07 47 AM](https://github.com/red-stone-network/clockwork/assets/91738741/21bac014-1b6a-4d22-b715-650588768365)

**Comment by Candy-Giants** [comment]
I agree that having an alternative app store would be convenient. Perhaps it could be added to the settings menu. However, I recall the "Read me" instructions warning against complaining about blocked content and suggesting to use the discussion tab instead for any concerns.

**Comment by lukasexists** [comment]
app store doesn't use that. i have plans for how to fix problems like these BUT i'm not bothered to work on clockwork atm. leaving this open so i can do it eventually

**Comment by orphanlol** [comment]
what

**Comment by NXX99** [comment]
> app store doesn't use vercel. i have plans for how to fix problems like these BUT i'm not bothered to work on clockwork atm. leaving this open so i can do it eventually. in the meantime you can probably ask around in the discord for app links from developers
> 
> @orphanlol please do not contact me on discord for feature requests, post them on here or in the discord

Also, it blocks anything with ".me" as the top level domain as well

**Comment by lukasexists** [comment]
I know this is really late to give an update BUT if you have a proxy there's a setting that'll route all your app through that, instructions can be found on the [wiki](https://github.com/red-stone-network/clockwork/wiki). I do have an idea for an alternate store that might be implemented if I get the motivation to work on Clockwork again, but until then there's not a ton that can be done.

**Comment by NXX99** [comment]
My school killed my ability to use proxies

**Comment by ToasterPanic** [comment]
Alternate stores will likely not be added in v2, though it will be added in v3 once that is started (which will be a while most likely.)

The store being blocked for some users is being discussed in #71.

Closing.
